.github/workflows/build.yml

```diff
@@ -13,16 +13,19 @@ on:
         default: 'false'
 
 env:
-  version: m110-ad42464-1
+  version: m110-d88a7b5
 
 jobs:
   macos:
-    runs-on: macos-11
+    runs-on: macos-13
```

Comment:
Does this change mean that skia/skiko/Compose now requires macOS minimum 13?

If so, let's keep 11, and if tvos requires 13, make it as a separate job.

Reply:
With macOS 11, it couldn't find some symbols during build with Apple TV SDK, like [MTLGPUFamilyApple6](https://developer.apple.com/documentation/metal/mtlgpufamily/mtlgpufamilyapple6) but I didn't dig deeper because I was solving a different problem with built and just used macOS 13, which I tested locally. I will investigate the requirements and revert iOS build to macOS 11 and see if tvOS can be built on it, or if we need a separate job. Thanks for noticing!